server/main.go

```diff
@@ -240,6 +240,10 @@ type mediaConfig struct {
 	Handlers map[string]json.RawMessage `json:"handlers"`
 }
 
+type LinkPreviewConfig struct {
```

Comment:
1. There is no need for a `struct` with just one field. Unless there are more config fields *now*, just use a boolean directly. 
2. You are exporting the type for no reason again. Please use exported types only if there is a need to export them.